{"name": "test_invalid_sign_in_get_request", "status": "passed", "steps": [{"name": "Fail sign in get request", "status": "passed", "start": 1648797904140, "stop": 1648797904352}], "start": 1648797904140, "stop": 1648797904353, "uuid": "b9f3a13a-3c86-4156-92bf-40ffb961ad60", "historyId": "b76b91d68b162f208ab544833704df5e", "testCaseId": "c03d7c083b451bb970c52430f0e4a590", "fullName": "tests.api_testcases.login_test.login_test.TestUserLogin#test_invalid_sign_in_get_request", "labels": [{"name": "parentSuite", "value": "tests.api_testcases.login_test"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "pop-os"}, {"name": "thread", "value": "105835-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testcases.login_test.login_test"}]}